dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -896,6 +896,7 @@
         #
         #     numberOfShards: 3
         #     numberOfReplicas: 0
+        #     templatePriority: 100
```

Comment:
❓ should this not be the default of 20 then?

Reply:
interesting, I thought we always aimed to match these templates up to the default values. But all good, no big deal 👍